docs/topics/mta-cli-transform.adoc

```diff
@@ -4,9 +4,14 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="mta-cli-transform_{context}"]
-= Performing transformation using the command line
+= Performing transformation by using the command line 
 
-Transform has two subcommands - `openrewrite` and `rules`.
+You can use transformation to perform the following actions:
+
+* Transform Java applications source code by using the `transform openrewrite` command.
+* Convert XML rules to YAML rules by using the `transform rules` command.
+
+IMPORTANT: Performing transformation requires configured container runtime. For more information, see xref:installing-using-podman_cli-guide[Installing the CLI by using Podman].
```

Comment:
This is a bit of a weird double modal - “requires configured” is nonstandard

```suggestion
IMPORTANT: Performing transformation requires a configured container runtime. For more information, see xref:installing-using-podman_cli-guide[Installing the CLI by using Podman].
```

Not sure this is suitable